mantle/cmd/kola/resources/iscsi_butane_setup.yaml

```diff
@@ -129,6 +128,8 @@ systemd:
         Type=oneshot
         RemainAfterExit=yes
         ExecStart=/usr/local/bin/targetcli_script
+        Restart=on-failure
```

Comment:
Do we still need this even with the healthcheck?

Reply:
yes, I found that on aarch64 this script requires two runs to work , the first attempt fails. 
Maybe the healthcheck does not work the way I expect it to